### PR TITLE
[Overview] 🌐 🐛 Fix GB Incorrect Values and Better Translations

### PR DESCRIPTION
**Pull request by Gooochy**

### CODE REVIEW

File: src/i18n/en_GB.ts [hunk 1]
```diff
@@ -846,7 +846,7 @@ export default {
             credits: 0,
             extensions: [],
             levelcost: [],
-            maxBuildings: 'All 25 buildings one control center',
+            maxBuildings: 'For every 25 buildings, you get 1 control center',
```
Comment (jxn-30) [inline src/i18n/en_GB.ts:849]: how about `, you can build one control center`?

File: src/i18n/en_GB.ts [hunk 2]
```diff
@@ -718,7 +718,7 @@ export default {
             maxBuildings: 'No limit',
             maxLevel: 0,
             special:
-                "Finance ministers and admins can (expand) association rescue schools with the help of credits from the association's treasury.Training course masters and admins can start training courses at association rescue schools.",
+                "Finance ministers and admins can (expand) association rescue schools with the help of credits from the association's funds .Alliance Educators and admins can start training courses at association rescue schools.",
```
Comment (jxn-30) [inline src/i18n/en_GB.ts:721]: switch ` ` and `.` to `. ` instead of ` .`

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by Gooochy** [comment]
Done it

File: src/i18n/en_GB.ts [hunk 3]
```diff
@@ -718,7 +718,7 @@ export default {
             maxBuildings: 'No limit',
             maxLevel: 0,
             special:
-                "Finance ministers and admins can (expand) association rescue schools with the help of credits from the association's treasury.Training course masters and admins can start training courses at association rescue schools.",
+                "Finance ministers and admins can (expand) association rescue schools with the help of credits from the association's funds . Alliance Educators and admins can start training courses at association rescue schools.",
```
Comment (jxn-30) [inline src/i18n/en_GB.ts:721]: now there's a space to much before the `.`

File: src/i18n/en_GB.ts [hunk 4]
```diff
@@ -846,7 +846,7 @@ export default {
             credits: 0,
             extensions: [],
             levelcost: [],
-            maxBuildings: 'All 25 buildings one control center',
+            maxBuildings: 'For every 25 buildings, , you can build one control center',
```
Comment (jxn-30) [inline src/i18n/en_GB.ts:849]: why double comma? Plz remove one comma + space

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**